src/main/kotlin/rys/ajaxpetproject/service/impl/ChatServiceImpl.kt

```diff
@@ -0,0 +1,38 @@
+package rys.ajaxpetproject.service.impl
+
+
+import org.bson.types.ObjectId
+import org.springframework.stereotype.Service
+import rys.ajaxpetproject.exception.ChatNotFoundException
+import rys.ajaxpetproject.exception.ChatsNotFoundException
+import rys.ajaxpetproject.model.MongoChat
+import rys.ajaxpetproject.repository.ChatRepository
+import rys.ajaxpetproject.service.ChatService
+
+@Service
+class ChatServiceImpl(val chatRepository: ChatRepository) : ChatService {
+
+    override fun createChat(mongoChat: MongoChat) = chatRepository.save(mongoChat)
+
+    override fun findChatById(id: ObjectId): MongoChat? = chatRepository.findChatById(id)
+
+    fun getChatById(id: ObjectId) = chatRepository.findChatById(id) ?: throw ChatNotFoundException()
+
+    fun getAllChats(): List<MongoChat> = chatRepository.findAllBy() ?: throw ChatsNotFoundException()
+
+    override fun findAllChats(): List<MongoChat>? = chatRepository.findAllBy() ?: emptyList()
+
+    override fun updateChat(id: ObjectId, updatedMongoChat: MongoChat) =
```

Comment:
please be consistent in your code. In some methods, you specify a return type, in some not. use one approach. the general rule is that public methods should have explicitly specified return type